api/anilist/src/main/kotlin/com/imashnake/animite/api/anilist/sanitize/media/Media.kt

```diff
@@ -0,0 +1,136 @@
+package com.imashnake.animite.api.anilist.sanitize.media
+
+import android.graphics.Color
+import com.imashnake.animite.api.anilist.MediaListQuery
+import com.imashnake.animite.api.anilist.MediaQuery
+import com.imashnake.animite.api.anilist.type.MediaRankType
+
+class Media(
+    /** @see MediaQuery.Media.bannerImage */
+    val bannerImage: String?,
+    /** @see MediaQuery.Media.coverImage */
+    val coverImage: String?,
+    /** @see MediaQuery.CoverImage.color */
+    val color: Int,
+    /** @see MediaQuery.Media.title */
+    val title: String?,
+    /** TODO: https://github.com/imashnake0/Animite/issues/58.
+     * @see MediaQuery.Media.description */
+    val description: String,
+    /** @see MediaQuery.Media.rankings */
+    val rankings: List<Ranking>,
+    /** @see MediaQuery.Media.genres */
+    val genres: List<String>,
+    /** @see MediaQuery.Media.characters */
+    val characters: List<Character>,
+    /** @see MediaQuery.Media.trailer */
+    val trailer: Trailer
+) {
+    data class Ranking(
+        /** @see MediaQuery.Ranking.rank */
+        val rank: Int,
+        /** @see MediaQuery.Ranking.type */
+        val type: Type,
+    ) {
+        /** @see MediaQuery.Ranking.type */
+        enum class Type(val string: String) {
+            RATED("Rated"),
+            POPULAR("Popular"),
+            SCORE("Score")
+        }
+    }
+
+    data class Character(
+        /** @see MediaQuery.Node.id */
+        val id: Int,
+        /** @see MediaQuery.Node.image */
+        val image: String?,
+        /** @see MediaQuery.Node.name */
+        val name: String?,
+    )
+
+    data class Trailer(
+        /** @see MediaQuery.Trailer.id
+         * @see MediaQuery.Trailer.site */
+        val url: String?,
+        /** @see MediaQuery.Trailer.thumbnail */
+        val thumbnail: String?,
+    ) {
+        /** @see MediaQuery.Trailer.thumbnail */
+        enum class Site(val baseUrl: String) {
+            YOUTUBE("https://www.youtube.com/watch?v="),
+            DAILYMOTION("https://www.dailymotion.com/video/"),
+            UNKNOWN("")
+        }
+    }
+
+    companion object {
+        fun sanitize(query: MediaQuery.Media) =
```

Comment:
Maybe this can be internal so we don't accidentally have usages outside of `api.anilist`?
```suggestion
        internal fun sanitize(query: MediaQuery.Media) =
```